projects/demo/src/modules/decorators/required-setter/required-setter-demo.component.ts

```diff
@@ -14,8 +14,8 @@ export class ExampleTuiRequiredSetterDemoComponent {
         quantity => Number.isInteger(quantity) && quantity >= 5,
         'Should be integer number more than min value',
     )
-    set quantity(quantity: number) {
-        this.items = new Array(quantity).fill(
+    set quantity(quantity: number | undefined) {
+        this.items = new Array(quantity ?? 0).fill(
```

Comment:
Same here. Looks like required setter decorator is completely broken and doesn't do anything 😱 I will look into it!